Sources/Settings/Controller/AboutController.swift

```diff
@@ -72,7 +72,6 @@ class AboutController: UIViewController {
 
     private func setupNavigationBar() {
         self.navigationItem.titleView = UIImageView(image: UIImage(named: "AboutTitle"))
-        self.navigationItem.titleView?.tintColor = PresentationTheme.current.colors.navigationbarTextColor
```

Comment:
with this line of code title color change had a lag (without manual change `largeTitleTextAttributes` - but it is not expected here, right?). Looks good even without this line of code.